lib/DDG/Spice/ExpandURL.pm

```diff
@@ -6,7 +6,7 @@ use DDG::Spice;
 
 triggers start => "http", "https", "expand", "expandurl", "unshorten";
 
-spice to => 'http://api.longurl.org/v2/expand?url=$1&format=json&callback={{callback}}';
+spice to => 'http://expandurl.com/api/v1/?url=$1&format=json';
```

Comment:
Try it out :smile: it sounds promising per the doc.

http://docs.duckduckhack.com/backend-reference/spice-attributes.html